### PR TITLE
added tests for exists unit-tests

### PR DESCRIPTION
**Pull request by ohadsahar**

### CODE REVIEW

File: __tests__/Task.test.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    };
+    const listName = TaskStatusType.CREATED;
+
+    render(
+      <AppTestUtil>
+        <Task
+          task={task}
+          setCurrentEdit={() => {}}
+          showCheck={true}
+          showPause={[
```
Comment (amitmarko) [inline __tests__/Task.test.tsx]: If the listName is created why we need to check?
```suggestion
          showPause
```

File: __tests__/Task.test.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      status: TaskStatusType.CREATED,
+    };
+    const listName = TaskStatusType.CREATED;
+
+    render(
+      <AppTestUtil>
+        <Task
+          task={task}
+          setCurrentEdit={() => {}}
+          showCheck={true}
```
Comment (amitmarko) [inline __tests__/Task.test.tsx]:
```suggestion
          showCheck
```

**Review by amitmarko — APPROVED** [review]
Small comment

File: __tests__/Task.test.tsx (hunk 2 above)
Comment (ohadsahar) [reply __tests__/Task.test.tsx]: Fixed

**Pull request merged**